client/components/build-list.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react'
+import Menu from '@material-ui/core/Menu'
+import MenuItem from '@material-ui/core/MenuItem'
+import Button from '@material-ui/core/Button'
+
+const styles = {
+  menuItem: {
+    width: '200px'
+  },
+  root: {
+    position: 'absolute',
+    top: '20px',
+    left: '70%'
+  }
+}
+
+export default class BuildList extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      anchorEl: null,
+      builds: []
+    }
+  }
+
+  componentDidMount() {
+    const req = {
+      method: 'GET',
+      headers: { 'Content-Type': 'application/json' }
+    }
+
+    fetch('computer-parts/save/builds', req)
+      .then(res => res.ok && res.json())
+      .then(builds => this.setState({ builds: builds }))
+      .catch(err => console.error(err))
+  }
+
+  handleClick = event => {
+    this.setState({ anchorEl: event.currentTarget })
```

Comment:
Why not set a `Boolean` on state? Is a `data-attribute` possible? This is not really something you do in React apps.